docs/application-baseline.md

```diff
@@ -1,26 +0,0 @@
----
```

Comment:
Was this doc relocated somewhere (docs repo)? Since this is specific to the contents of core I would think we want to keep it here, but might just be missing why it was removed/moved.

Reply:
Most of the content was moved here: https://uds.defenseunicorns.com/overview/uds-structure/#uds-core-capabilities. Some of the other non-table stuff might not exist so perhaps I just put this back. My bad if it's not anywhere else.